test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -181,18 +182,18 @@ u256 EVMInstructionInterpreter::eval(
 		return u256(keccak256(readMemory(offset, size)));
 	}
 	case Instruction::ADDRESS:
-		return m_state.address;
+		return h256(m_state.address, h256::AlignRight);
```

Comment:
Why are these not u256 to begin with?

Reply:
Good point. Tried now using u256 and it complicates code quite a bit for `EwasmBuiltinInterpreter`, and the fact this shows clearly the size of addresses, it may be a good option to go with?